chart/base/values.yaml

```diff
@@ -0,0 +1,40 @@
+global:
+  tlsSecretName: ""
+  ingressSubdomain: ""
+
+base:
+  runtime: python
+
+  image:
+    repository: mynamespace/myimage
```

Comment:
Did you mean to fill this in?

Reply:
That's a place holder, the user would need to provide the actual image that they build using the Dockerfile

When using the cloudnativetoolkit.dev pipelines this value is generated based on the image building.

Reply:
putting a better default image now